src/KMeans/index_test.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2019 ml5
+//
+// This software is released under the MIT License.
+// https://opensource.org/licenses/MIT
+
+const {
+    kmeans
+} = ml5;
+
+const KMEANS_DEFAULTS = {
+    'k': 2,
+    'maxIter': 5,
+    'threshold': 0.5,
+};
+
+describe('kMeans', () => {
+    let kmeansModel;
+    const dataurl = 'https://raw.githubusercontent.com/ml5js/ml5-examples/development/d3/KMeans/KMeans_GaussianClusterDemo/data/gaussian2d_2clusters.csv'
+    beforeEach(async () => {
+        jasmine.DEFAULT_TIMEOUT_INTERVAL = 10000;
+        kmeansModel = await kmeans(dataurl, KMEANS_DEFAULTS, (err, result) => {
+            return;
+        });
+        await kmeansModel.load(dataurl)
```

Comment:
Making a note on this: it seems that the kmeans class will not wait until the data is fully loaded and therefore we need to `await kmeansModel.load()` which is not what we want. I can take a look at why this is occurring.

cc/ @jwilber